src/main/java/gregtech/api/fluids/GTFluidBlock.java

```diff
@@ -19,19 +20,18 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-public class MaterialFluidBlock extends BlockFluidClassic {
+public class GTFluidBlock extends BlockFluidClassic {
 
-    private final Material gtMaterial;
```

Comment:
I don't see the point of removing this? I think it is for the ability to declare non-material fluids, but you could just make this material nullable, and return null for non-material fluids